javascript_nodejs/17.proactive-messages/dialogs/mainDialog/index.js

```diff
@@ -3,15 +3,19 @@
 
 const { ActivityTypes, TurnContext } = require('botbuilder');
 
+const JOBS_LIST = 'jobs';
+
 class MainDialog {
     /**
      * 
-     * @param {Storage} storage A storage system like MemoryStorage used to store information.
+     * @param {ConversationState} conversationState A ConversationState object used to track the dialog.
```

Comment:
This needs to change to use BotState; otherwise, the cross conversation, cross user scenario doesn't work.